src/test/java/com/activestate/cloudfoundryjenkins/DeploymentInfoTest.java

```diff
@@ -158,7 +158,7 @@ public void testReadJenkinsConfigDefaultOptions() throws Exception {
 
         assertEquals("jenkins-build-name", deploymentInfo.getAppName());
         assertEquals(DEFAULT_MEMORY, deploymentInfo.getMemory());
-        assertEquals("jenkins-build-name", deploymentInfo.getHostname());
```

Comment:
I think we can leave the Hostname as it is , since it was alrady existing field.  
We need tests for the new fields. In fact, where are the tests that lead to the new state vars in DepoloymentInfo?